src/wrap-store/wrapStore.js

```diff
@@ -44,7 +44,11 @@ export default (store, {
    */
   chrome.runtime.onMessage.addListener((request, sender, sendResponse) => {
     if (request.type === DISPATCH_TYPE) {
-      dispatchResponder(store.dispatch(request.payload), sendResponse);
+      const action = Object.assign({}, request.payload, {
+        _sender: sender,
```

Comment:
I hear you! I'm not against it in principle! Just want to stay consistent within the package :)